basic_pitch/saved_models/icassp_2022/nmp.mlpackage/Manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "fileFormatVersion": "1.0.0",
+    "itemInfoEntries": {
+        "7EB69391-53BD-4281-BFA1-831180579ABC": {
+            "author": "com.apple.CoreML",
+            "description": "CoreML Model Specification",
+            "name": "model.mlmodel",
+            "path": "com.apple.CoreML/model.mlmodel"
+        },
+        "C784FB56-1BA8-4A0E-AC2A-F0388FE1A2DB": {
+            "author": "com.apple.CoreML",
+            "description": "CoreML Model Weights",
+            "name": "weights",
+            "path": "com.apple.CoreML/weights"
+        }
+    },
+    "rootModelIdentifier": "7EB69391-53BD-4281-BFA1-831180579ABC"
+}
```

Comment:
what is the purpose of this file, or where does this way of specifying come from?

Reply:
It's a coreml file. `.mlpackage` is the extension for coreml files.